### PR TITLE
[release-2.7] ACM-2463 : adds mce-must-gather script

### PR DESCRIPTION
**Pull request by eemurphy**
Signed-off-by: Erin Murphy erinmurp@redhat.com

**Related Issue:**  https://issues.redhat.com/browse/ACM-2463

**Description of Changes:** Adds copy of must-gather from backplane-must-gather

**What resource is being added**: MCE

### CODE REVIEW

File: collection-scripts/mce-gather.sh [hunk 1]
```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+#set -x
+# Copyright (c) 2021 Red Hat, Inc.
```
Comment (jlpadilla) [inline collection-scripts/mce-gather.sh:3]:
```suggestion
# Copyright (c) 2023 Red Hat, Inc.
```

**Comment by jlpadilla** [comment]
/hold 
Will wait until we validate these changes on 2.8 before merging into 2.6 and 2.7

**Comment by eemurphy** [comment]
@jlpadilla could this and the 2.6 must-gather PRs be approved and merged? The 2.8 must-gather changes was looked into to see if canaries are taking longer and there isn't any significant change. Seems to not be affecting it, as expected!

**Comment by jlpadilla** [comment]
@eemurphy Thanks for verifying the canary. Currently `release-2.7` is on code freeze, will merge as soon as it reopens for 2.7.2

**Comment by jlpadilla** [comment]
Still waiting for release-2.7 to open for 2.7.2

**Comment by jlpadilla** [comment]
ACM 2.7.2 only included critical fixes and we missed the merge window. Planning to merge as soon as 2.7.3 is open.

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**